app/src/main/java/nu/yona/app/ui/dashboard/SingleDayActivityDetailFragment.java

```diff
@@ -298,6 +301,17 @@ private void initilizeCommentControl(View view)
 	}
 
 
+	private void setUpDayAcvitivy()
+	{
+		if (getArguments() != null)
+		{
+			if (getArguments().get(AppConstant.OBJECT) instanceof DayActivity)
```

Comment:
Don't nest if-statements unnecessarily. Nesting adds complexity.
As indicated in my earlier comment, this is sufficient:
```
if (getArguments() != null && getArguments().get(AppConstant.OBJECT) instanceof DayActivity)
```